tests/controller/user_controller/test_user_controller.py

```diff
@@ -1,2 +1,63 @@
+import pytest
+import json
+from src.controller.user_controller.user_controller import UserController
+from src.dto.user.user_dto import User
+import unittest.mock as mock
+from flask import jsonify, request
+from src.app import app
+import unittest
 
+class TestUserController(unittest.TestCase):
```

Comment:
@sweiyang , i managed to get unit tests and mocking to work. Thanks to @gitkh2891 given me some clues. 

Now pending me to fix sonarcloud. Don't know why it is not capturing the coverage.xml for sonar scanning

Reply:
https://community.sonarsource.com/t/failed-to-resolve-file-paths-in-coverage-xml-python-coverage-report-when-its-in-a-subdirectory/75560

![image](https://user-images.githubusercontent.com/10574652/233854851-5246fdaa-bdec-4167-a104-09c0706b8ba7.png)


ZZZzzzz

Reply:
![Screenshot 2023-04-24 at 1 45 54 AM](https://user-images.githubusercontent.com/10574652/233856065-b1f579d4-af9d-4be6-9246-1b88324e6b92.png)

Working :)